.github/workflows/snl-ci.yml

```diff
@@ -143,7 +143,8 @@ jobs:
           cmake -S . -B build \
             -DCMAKE_CXX_COMPILER=icpx \
             -DCMAKE_CXX_STANDARD=20 \
-            -DCMAKE_CXX_FLAGS='-Werror -fp-model=precise -fsycl-device-code-split=per_kernel -flink-huge-device-code -Wno-pass-failed' \
+            -DCMAKE_CXX_FLAGS='-Werror -fp-model=precise -fsycl-device-code-split=per_kernel -Wno-pass-failed' \
+            -DCMAKE_EXE_LINKER_FLAGS='-flink-huge-device-code' \
```

Comment:
```
  /opt/intel/oneapi/compiler/2024.2/bin/icpx -Werror -fp-model=precise
  -fsycl-device-code-split=per_kernel -Wno-pass-failed
  -flink-huge-device-code CMakeFiles/cmTC_c1874.dir/cplusplus17.cpp.o -o
  cmTC_c1874

  icpx: error: argument unused during compilation: '-flink-huge-device-code'
  [-Werror,-Wunused-command-line-argument]

Reply:
I put it in the wrong spot, thanks, lets try this